tools/pd-heartbeat-bench/main.go

```diff
@@ -411,6 +432,28 @@ func pick(slice []int, cfg *config.Config, ratio float64) []int {
 	return append(slice[:0:0], slice[0:int(float64(cfg.RegionCount)*ratio)]...)
 }
 
+func pickNoUpdatedRegions(slice []int, cfg *config.Config, ratio float64, updatedMap map[int]*pdpb.RegionHeartbeatRequest) []int {
+	if ratio == 0 {
+		return nil
+	}
+	rand.Shuffle(cfg.RegionCount, func(i, j int) {
+		slice[i], slice[j] = slice[j], slice[i]
+	})
+	NoUpdatedRegionsNum := int(float64(cfg.RegionCount) * ratio)
```

Comment:
And do we need check the `NoUpdatedRegionsNum + len(updatedMap)` should less than `len(slice)`